consensus/common/src/main/java/tech/pegasys/pantheon/consensus/common/ValidatorVotePolarity.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.consensus.common;
+
+// Determines if a validator vote is indicating that they should be added, or removed.
+// This does not attempt to determine how said vote should be serialised/deserialised.
```

Comment:
If we're going to write class comments, they may as well be in JavaDoc form.

Reply:
done.